forum/backends/mongodb/api.py

```diff
@@ -1581,19 +1581,25 @@ def get_course_id_by_thread_id(thread_id: str) -> str | None:
         """
         Return course_id for the matching thread.
         """
-        thread = CommentThread().get(thread_id)
-        if thread:
-            return thread.get("course_id")
+        try:
+            thread = CommentThread().get(thread_id)
+            if thread:
+                return thread.get("course_id")
+        except bason_errors.InvalidId:
```

Comment:
In which case(s) is this exception happening?

Reply:
Got it! Please add a comment :)

Reply:
Done